src/common/models/Projection.ts

```diff
@@ -180,6 +183,7 @@ export class Projection {
     this.dailyNegativeTests = this.deltasFromCumulatives(
       this.cumulativeNegativeTests,
     );
+    this.dailyDeaths = this.deltasFromCumulatives(this.cumulativeDeaths);
```

Comment:
this.cumulativeDeaths is defined above as:
```
    this.cumulativeDeaths = timeseries.map(row => row && row.cumulativeDeaths);
```

Confusingly, our API has `timeseries` and `actualTimeseries`.  `timeseries` is what comes out of our SEIR model and they are _estimates_.  `actualTimeseries` is the raw actual data we have from CovidTracking or Corona Data Scraper or our other data sources.

So in this case, we want to use the actual deaths, which is probably something like:
```
    const cumulativeActualDeaths = actualTimeseries.map(row => row?.cumulativeDeaths || null);
    this.dailyDeaths = this.deltasFromCumulatives(cumulativeActualDeaths);
```